RELEASE_NOTES.md

```diff
@@ -1,97 +1,11 @@
-# Chef Client Release Notes 12.6.0:
+# Chef Client Release Notes 12.7:
 
+## Updates to versioning strategy
```

Comment:
some words/thoughts to consider/merge.

We recently updated the versioning specification in our release process to facilitate faster releases with lower risk between each release. Soon an automated process will begin updating the "patch" version (the third number in the version).

For users of Chef, this means that the first release (ED: promotion builds are releases, but builds in current are still real and have versions) of 12.7 you use may be 12.7.2 or 12.7.5, without earlier versions of 12.7 having been released first. Those earlier versions may be available in the prerelease channel we call current that you can access through the install.sh and install.ps1 scripts. <words about semver>